modules/map/hdmap/adapter/xml_parser/status.h

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2017 The Apollo Authors. All Rights Reserved.
```

Comment:
There's already a same Status class implemented in modoules/common. Please remove this and use that instead.